dev/com.ibm.ws.wssecurity_fat.wsscxf.1/fat/src/com/ibm/ws/wssecurity/fat/cxf/x509migtoken/CxfX509MigTests.java

```diff
@@ -2939,6 +2939,7 @@ public void testBadCxfX509AsymIssuerSerialMigService() throws Exception {
      */
 
     @Test
+    @AllowedFFDC(value = { "org.apache.cxf.ws.policy.PolicyException" }, repeatAction = { JakartaEE9Action.ID })
```

Comment:
I'm concerned about these changes. It doesn't seem correct that xmlWS-3.0's tests generate one FFDC, when the other runs generate a different FFDC. Updating the the FFDC can be OK in certain situations, but I don't think this `@AllowedFFDC` should be necessary. I think we need to investigate why one is getting one version and the other is getting another.